pkgs/top-level/all-packages.nix

```diff
@@ -16722,6 +16722,16 @@ with pkgs;
 
   toxic = callPackage ../applications/networking/instant-messengers/toxic { };
 
+  tqsl = callPackage ../applications/misc/tqsl {
+    cmake = cmake;
+    expat = expat;
+    libssl = openssl;
+    zlib = zlib;
+    db = db;
+    curl = curl;
+    wxGTK = wxGTK;
```

Comment:
I don't think you need these? Also why not just name `libssl` `openssl`?

Reply:
I used libssl because I saw some other package do it.

You're right; I don't need those. I forgot they'd automatically be in scope.

Reply:
I see. I just did a quick check and it looks like only axel names openssl as libssl. :woman_shrugging: